OpenUpSA/wazimap-ng-ui
# fixing the logic error that hides the filter area - staging

## Description


## Related Issue
https://trello.com/c/B2MnfGmy/882-no-filters-shown-for-an-indicator-which-is-supposed-to-have-a-filter

## How to test it locally
* Go to Sanef profile
* Expand the data mapper 
* Financial performance > Income sources > % of income by financial year > 2016-2017
* Confirm that the filter area in the mapchip is visible

## Screenshots
![image](https://user-images.githubusercontent.com/53019884/128859288-659e54cb-c2a5-4656-bfea-0e043e35999b.png)


## Changelog

### Added

### Updated

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
